Binary files imx-bootlets-src-10.12.01/boot_prep/.init-mx23.c.swp and imx-bootlets-src-10.12.01-modified/boot_prep/.init-mx23.c.swp differ
diff -urN imx-bootlets-src-10.12.01/linux_prep/cmdlines/stmp378x_dev.txt imx-bootlets-src-10.12.01-modified/linux_prep/cmdlines/stmp378x_dev.txt
--- imx-bootlets-src-10.12.01/linux_prep/cmdlines/stmp378x_dev.txt	2012-10-06 15:02:38.691094106 +0900
+++ imx-bootlets-src-10.12.01-modified/linux_prep/cmdlines/stmp378x_dev.txt	2012-12-23 21:57:26.816233531 +0900
@@ -1,3 +1,4 @@
-console=ttyAMA0,115200 root=/dev/mmcblk0p2 rw rootwait ssp1=mmc lcd_panel=lms430 no_console_suspend
-console=ttyAMA0,115200 root=/dev/mmcblk0p2 rw rootwait ssp1=mmc lcd_panel=lms350
-console=ttyAMA0,115200 ssp1=spi1 ubi.mtd=2 root=ubi0:rootfs0 rootfstype=ubifs lcd_panel=lms430
+console=ttyAMA0,115200 ssp1=mmc ubi.mtd=2 root=ubi0:rootfs rootfstype=ubifs lcd_panel=tm024hbh40
+console=ttyAMA0,115200 ssp1=mmc ubi.mtd=2 root=ubi0_0 rootfstype=ubifs lcd_panel=tm024hbh40
+console=ttyAMA0,115200 ssp1=mmc ubi.mtd=2 root=ubi0_0 rootfstype=ubifs lcd_panel=tm024hbh40
+
